src/tools/dev/scripts/bv_support/bv_gdal.sh

```diff
@@ -137,18 +137,18 @@ EOF
 function build_gdal
 {
     #
-    # Prepare build dir
+    # Uncompress the source file
     #
-    prepare_build_dir $GDAL_BUILD_DIR $GDAL_FILE
+    uncompress_src_file $GDAL_SRC_DIR $GDAL_FILE
     untarred_gdal=$?
     if [[ $untarred_gdal == -1 ]] ; then
-        warn "Unable to prepare GDAL Build Directory. Giving Up"
+        warn "Unable to uncompress GDAL Build Directory. Giving Up"
```

Comment:
Um, nope...here is another case.

Reply:
The warning still mentions 'Build Directory' here.